rviz_common/src/rviz_common/properties/property_tree_widget.cpp

```diff
@@ -101,7 +101,7 @@ void PropertyTreeWidget::setModel(PropertyTreeModel * model)
   if (model_) {
     connect(
       model_, SIGNAL(propertyHiddenChanged(const Property*)),
-      this, SLOT(propertyHiddenChanged(const Property*)), Qt::QueuedConnection);
+      this, SLOT(propertyHiddenChanged(const Property*)));
```

Comment:
With the (rather large) caveat that I'm not super familiar with Qt's event loop, this seems like it has the same problem.  That is, omitting this ends up using the "automatic" connection type from Qt: https://doc.qt.io/qt-5/qt.html#ConnectionType-enum .  If the automatic connection type decides to use a queued connection, then you'll still have the same problem.

Should we instead make this a `DirectConnection`?